client/components/Auth/AuthInfo.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import Link from "next/link";
+
+interface AuthInfoProps {
+	heading: string;
+	text: string;
+	link?: string;
+	linkText?: string;
+}
+
+const AuthInfo = ({ heading, text, link, linkText }: AuthInfoProps) => {
+	return (
+		<div className="auth__info">
+			<h3>{heading}</h3>
+			<p>
```

Comment:
might as well create a component for 'p' to control it dynamically with different children